src/scss/_mixins.scss

```diff
@@ -0,0 +1,61 @@
+@mixin appearance($value) {
```

Comment:
It's worth noting that we went from `mixins/_index.scss` to `_mixins.scss` to simplify the `@import` statement from 

```scss
@import "mixins/index";
```

to 

```scss
@import "mixins";
```